package.json

```diff
@@ -92,6 +94,7 @@
     "@nrwl/angular": "^9.1.0",
     "@nrwl/jest": "^9.1.0",
     "@nrwl/node": "^9.1.0",
+    "@nrwl/web": "^9.1.0",
```

Comment:
Do we need it?

Reply:
I don't think we need it... I probably added it during a testing phase, I will remove it 👍